app/extensions/brave/content/scripts/passwordManager.js

```diff
@@ -45,11 +45,10 @@ if (chrome.contentSettings.passwordManager == 'allow') {
     }
 
     if (credentials[action]) {
-      credentials[action].push([passwordElem, usernameElem])
-    } else {
-      credentials[action] = [[passwordElem, usernameElem]]
+      return
```

Comment:
ah, I assumed each action had a single username/password element. The purpose of this change was to avoid adding another listener on each check so just watch out for that with the fix